packages/wrangler/package.json

```diff
@@ -119,7 +119,7 @@
     "bundle": "node -r esbuild-register scripts/bundle.ts",
     "build": "npm run clean && npm run bundle",
     "prepublishOnly": "npm run build",
-    "start": "npm run bundle && NODE_OPTIONS=--enable-source-maps ./bin/wrangler.js",
+    "start": "SOURCEMAPS=true npm run bundle && NODE_OPTIONS=--enable-source-maps ./bin/wrangler.js",
```

Comment:
I don't find `npm start` very helpful when debugging as the working directory seems to get messed up.

So I tend to just do:

```
npm run build
cd some/folder
npx wrangler
```

This would mean that I no longer get source-maps for this flow.

Reply:
Does the `prepublishOnly` script get run before we actually publish to npm?

If so, how about we reverse this. Always generate source-maps unless turned off. Then turn it off in the `prepublishOnly` script?

Reply:
done, let's see what CI says

Reply:
hrmm doesn't look like that worked, investigating

Reply:
Oh, it did work, but prereleases run the plain build, which includes sourcemaps. That's fine I guess.